Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -1154,6 +1154,53 @@ extension NSManagedObjectContext: MessageDatabaseSession {
             $0.localState = .pendingUpload
         }
     }
+    
+    func loadMessage(
```

Comment:
Moved these under DatabaseSession so that I can use DatabaseSession in async version of the read()